crates/egui/src/containers/scroll_area.rs

```diff
@@ -54,6 +54,11 @@ impl State {
     pub fn store(self, ctx: &Context, id: Id) {
         ctx.data_mut(|d| d.insert_persisted(id, self));
     }
+
+    /// Returns true if the scroll area currently has a velocity due to momentum.
+    pub fn has_velocity(&self) -> bool {
+        self.vel.x != 0.0 || self.vel.y != 0.0
```

Comment:
We could also just return the current velocity, which gives the user a lot more info